db/migrate/20150706184439_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
+      t.string   :slug,           :null => false
+      t.integer  :sluggable_id,   :null => false
+      t.string   :sluggable_type, :limit => 50
+      t.string   :scope
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.